packages/paste-core/components/pagination/src/PaginationArrow/PaginationArrowSVG.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+
+const PaginationArrowSVG: React.FC = ({children}) => {
```

Comment:
I don't _love_ this component name. This isn't quite an SVG, but a wrapper around the SVG Icon.

Reply:
This could also be rewritten as
```js
export const PaginationArrowIconWrapper = styled.div({
  css({
    alignItems: 'center',
    ...
   })
})
```

Reply:
We should try to avoid styled elements where possible going forward. `Box` all the way

Reply:
I like the name change idea. Good call.